### PR TITLE
A4A: add woo product-filters data

### PR DESCRIPTION
**Pull request by andrii-lysenko**
Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/882

## Proposed Changes

As a part of adding new product we need to make sure we have proper product name and description within A4A. This PR adds description to the A4A marketplace as well as Product lighbox:

**Marketplace**
Screenshot 2024-10-29 at 6 19 39 PM

**Lighbox**
Screenshot 2024-10-29 at 6 19 31 PM

**Checkout**
Screenshot 2024-10-29 at 6 19 52 PM

## Testing Instructions

- Change billing scheme. `wp jetpack-start partner-key-billing-scheme-override add [Partner Key ID] 877` 
- Navigate to `agencies.automattic.com/marketplace/products` and check that you see the Woo Product Filters product. Check the descriptions and purchase flow.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

**Review by jkguidaven — APPROVED** [review]
Works as expected. LGTM 👍

File: client/jetpack-cloud/sections/partner-portal/hooks/index.tsx [hunk 1]
```diff
@@ -413,6 +413,11 @@ export function useProductDescription( productSlug: string ): {
 					'The only payment solution fully integrated to Woo. Accept credit/debit cards and local payment options with no setup or monthly fees.'
 				);
 				break;
+			case 'woocommerce-product-filters':
+				description = translate(
+					'This is a tool to create ajax product filters that make the process of finding products in your store simple and fast.'
```
Comment (vitozev) [inline client/jetpack-cloud/sections/partner-portal/hooks/index.tsx:418]: I think `ajax` here should be capitalized since it's an acronym?

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16955310

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @andrii-lysenko for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16955310).

**Pull request merged**